### ISSUE TITLE
[question]: why hoodieAccount service does not expose hoodie account api completely?

### ISSUE THREAD

**Issue by fredguth** [hoodiehq/ember-hoodie#74]
In [hoodie-account.js](https://github.com/hoodiehq/ember-hoodie/blob/master/addon/services/hoodie-account.js), the service does not allow:
account.validate
~~account.signUp~~
~~account.signIn~~
~~account.signOut~~
~~account.destroy~~
account.get
account.update
account.profile.get
account.profile.fetch
account.profile.update
account.request
account.on
account.one
account.off

is there a reason for that?  It doesn't seem right.

**Comment by gr2m** [comment]
I agree, it should expose all of the account methods. Note that there was a change recently: https://github.com/hoodiehq/hoodie-account-client/releases/tag/v6.0.0

I think `account.profile.fetch` no longer exists, also the `account.ready` method was removed. It would be great if you could help us make `ember-hoodie` up to date. I used it myself in the past but am not right now so can’t help maintain it at this point. Happy to onboard you :)

**Comment by fredguth** [comment]
I will work on this next week,  I was already planning to take a deeper look in ember-hoodie.
